pkgs/development/tools/misc/arcanist/default.nix

```diff
@@ -1,53 +1,48 @@
-{ stdenv, fetchFromGitHub, php, flex, makeWrapper }:
+{ stdenv, fetchFromGitHub, php, flex }:
+
+# Make a custom wrapper. If `wrapProgram` is used, arcanist thinks .arc-wrapped is being
+# invoked and complains about it being an unknown toolset. We could use `makeWrapper`, but
+# then we’d need to still craft a script that does the `php libexec/arcanist/bin/...` dance
+# anyway... So just do everything at once.
+let makeWrapper = toolset: ''
```

Comment:
Nit: rename to `makeArcWrapper` or something like this, so as not to risk confusing with the usual `makeWrapper`